tests/h/search/core_test.py

```diff
@@ -19,14 +19,18 @@
 class TestSearch(object):
     """Unit tests for search.Search when no separate_replies argument is given."""
     def test_it_defaults_separate_wildcard_uri_keys_to_true(self, pyramid_request, UriCombinedWildcardFilter):
+        separate_keys = True
+
         search.Search(pyramid_request)
 
-        UriCombinedWildcardFilter.assert_called_once_with(pyramid_request, True)
+        UriCombinedWildcardFilter.assert_called_once_with(pyramid_request, separate_keys)
```

Comment:
This is fine, though I was actually thinking of a kwarg as in: `assert_called_once_with(pyramid_request, separate_keys=True)`.